llvm/test/CodeGen/AArch64/smul_fix.ll

```diff
@@ -0,0 +1,93 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=aarch64-linux-gnu | FileCheck %s
+
+declare  i4  @llvm.smul.fix.i4   (i4,  i4, i32)
+declare  i32 @llvm.smul.fix.i32  (i32, i32, i32)
+declare  i64 @llvm.smul.fix.i64  (i64, i64, i32)
```

Comment:
The `declare`  is not necessary anymore:
https://discourse.llvm.org/t/recent-improvements-to-the-ir-parser/77366

Reply:
Done